src/pages/reader.vue

```diff
@@ -116,9 +163,68 @@ export default {
       } else {
         this.connectWallet();
       }
+
+      if (this.format === 'epub') {
+        this.initEpubRendition();
+      }
     } finally {
       this.isLoading = false;
     }
   },
+  methods: {
+    initEpubRendition() {
+      this.epubBook = Epub(this.fileSrc);
+      this.epubBook.loaded.navigation.then(
+        navigation => (this.epubTOC = navigation.toc)
+      );
+      this.epubRendition = this.epubBook.renderTo('viewer', {
+        width: '100%',
+        height: '100%',
+        spread: 'always',
+      });
+      this.epubRendition.display();
+      this.epubRendition.on('rendered', () => {
+        this.epubSelectedChapter = this.epubRendition.currentLocation().start.href;
+      });
+    },
+    onChangeEpubChapter() {
+      this.epubRendition.display(this.epubSelectedChapter);
+    },
+    onClickEpubPrev() {
+      this.epubRendition.prev();
+    },
+    onClickEpubNext() {
+      this.epubRendition.next();
+    },
+    async onClickDownloadEpub() {
+      try {
+        this.alertPromptSuccess(this.$t('nft_download_content_prepare'));
+        const blob = await this.$axios.$get(this.fileSrc, {
+          responseType: 'blob',
+        });
+        saveAs(blob, getDownloadFilenameFromURL(this.fileSrc));
+      } catch (error) {
+        // eslint-disable-next-line no-console
+        console.error(error);
+        this.alertPromptError(this.$t('nft_download_content_error'));
+      }
+    },
+  },
 };
 </script>
+
+<style>
+#viewer:after {
+  position: absolute;
+  width: 1px;
+  border-right: 1px #000 solid;
+  height: 90%;
+  z-index: 1;
+  left: 50%;
+  margin-left: -1px;
+  top: 5%;
+  opacity: 0.15;
+  box-shadow: -2px 0 15px rgba(0, 0, 0, 1);
+  content: '';
+}
+</style>
```

Comment:
not sure if it is better to have separate pages for PDF and Epub readers before the logic and styling are intertwined tgt

e.g.
```js
// Before
'/reader'

// After
'/reader/pdf'
'/reader/epub'
```